### PR TITLE
add cn doc of dropout series

### PR DESCRIPTION
**Pull request by huangjun12**
Add cn_doc of dropout series:
(1) paddle.nn.functional.dropout(x, p=0.5, axis=None, training=True, mode="upscale_in_train”, name=None)
(2) paddle.nn.functional.dropout2d(x, p=0.5, training=True, name=None)
(3) paddle.nn.functional.dropout3d(x, p=0.5, training=True, name=None)
(4) paddle.nn.functional.alpha_dropout(x, p=0.5, training=True, name=None)
(5) paddle.nn.Dropout(p=0.5, axis=None, mode="upscale_in_train”, name=None)
(6) paddle.nn.Dropout2D(p=0.5, data_format='NCHW', name=None)
(7) paddle.nn.Dropout3D(p=0.5, data_format='NCDHW', name=None)
(8) paddle.nn.AlphaDropout(p=0.5, name=None)

### CODE REVIEW

**Comment by huangjun12** [comment]
文档预览链接: http://180.76.141.178/documentation/docs/zh/api/index_cn.html?reviewVersion=jenkins-doc_review-753

**Comment by huangjun12** [comment]
image
image
image
image
image
image

**Comment by huangjun12** [comment]
image
image
image
image
image
image
image
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image
image

**Comment by huangjun12** [comment]
image
image
image

**Comment by huangjun12** [comment]
image
image
image

**Comment by huangjun12** [comment]
image
image
image

**Review by iclementine — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/nn_cn/Dropout2D_cn.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+在动态图模式下，请使用模型的 `eval()` 方法切换至测试阶段。
+
+.. note::
+   对应的 `functional方法` 请参考: :ref:`cn_api_nn_functional_dropout2d` 。
+
+参数
+:::::::::
+ - **p** (float): 将输入通道置0的概率， 即丢弃概率。默认: 0.5。
+ - **name** (str，可选): 操作的名称(可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+ - **data_format** (str): 指定输入的数据格式，输出的数据格式将与输入保持一致，可以是 `NCHW` 和 `NHWC` 。其中 `N` 是批尺寸， `C` 是通道数， `H` 是特征高度， `W` 是特征宽度。默认值: `NCHW` 。
```
Comment (jzhang533) [inline doc/fluid/api_cn/nn_cn/Dropout2D_cn.rst:18]: 调一下顺序

File: doc/fluid/api_cn/nn_cn/Dropout3D_cn.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+在动态图模式下，请使用模型的 `eval()` 方法切换至测试阶段。
+
+.. note::
+   对应的 `functional方法` 请参考: :ref:`cn_api_nn_functional_dropout3d` 。
+
+参数
+:::::::::
+ - **p** (float): 将输入通道置0的概率， 即丢弃概率。默认: 0.5。
+ - **name** (str，可选): 操作的名称(可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
```
Comment (jzhang533) [inline doc/fluid/api_cn/nn_cn/Dropout3D_cn.rst]: 调一下顺序

File: doc/fluid/api_cn/nn_cn/Dropout_cn.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+.. note::
+   对应的 `functional方法` 请参考: :ref:`cn_api_nn_functional_dropout` 。
+
+参数
+:::::::::
+ - **p** (float): 将输入节点置为0的概率， 即丢弃概率。默认: 0.5。
+ - **axis** (int|list): 指定对输入 `Tensor` 进行Dropout操作的轴。默认: None。
+ - **name** (str，可选): 操作的名称(可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+ - **mode** (str): 丢弃单元的方式，有两种'upscale_in_train'和'downscale_in_infer'，默认: 'upscale_in_train'。计算方法如下:
```
Comment (jzhang533) [inline doc/fluid/api_cn/nn_cn/Dropout_cn.rst:19]: 顺序

Comment (huangjun12) [reply doc/fluid/api_cn/nn_cn/Dropout_cn.rst:19]: Done

File: doc/fluid/api_cn/nn_cn/Dropout3D_cn.rst (hunk 2 above)
Comment (huangjun12) [reply doc/fluid/api_cn/nn_cn/Dropout3D_cn.rst]: Done

File: doc/fluid/api_cn/nn_cn/Dropout2D_cn.rst (hunk 1 above)
Comment (huangjun12) [reply doc/fluid/api_cn/nn_cn/Dropout2D_cn.rst:18]: Done

File: doc/fluid/api_cn/nn_cn/Dropout2D_cn.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+参数
+:::::::::
+ - **p** (float): 将输入通道置0的概率， 即丢弃概率。默认: 0.5。
+ - **data_format** (str): 指定输入的数据格式，输出的数据格式将与输入保持一致，可以是 `NCHW` 和 `NHWC` 。其中 `N` 是批尺寸， `C` 是通道数， `H` 是特征高度， `W` 是特征宽度。默认值: `NCHW` 。
+ - **name** (str，可选): 操作的名称(可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+
+形状
+:::::::::
+ - **输入** : 4D `Tensor` 。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Dropout2D_cn.rst]: 这里说法应该是 4-D Tensor 下同

File: doc/fluid/api_cn/nn_cn/Dropout_cn.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+       - train: out = input * mask
+       - inference: out = input * (1.0 - p)
+
+ - **name** (str，可选): 操作的名称(可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+
+形状
+:::::::::
+ - **输入** : N维 `Tensor` 。
+ - **输出** : N维 `Tensor` ，形状与输入相同。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Dropout_cn.rst]: N-D Tensor

**Review by jzhang533 — APPROVED** [review]
lgtm
